clients/samba/hive.yaml

```
@@ -0,0 +1,2 @@
+roles:
+  - samba
```

```suggestion
  - portal
```

this should be portal